homeassistant/components/garmin_connect/const.py

```diff
@@ -136,8 +140,8 @@
         None,
         False,
     ],
-    "averageStressLevel": ["Avg Stress Level", "", "mdi:flash-alert", None, True],
-    "maxStressLevel": ["Max Stress Level", "", "mdi:flash-alert", None, True],
+    "averageStressLevel": ["Avg Stress Level", "lvl", "mdi:flash-alert", None, True],
+    "maxStressLevel": ["Max Stress Level", "lvl", "mdi:flash-alert", None, True],
     "stressQualifier": ["Stress Qualifier", "", "mdi:flash-alert", None, False],
```

Comment:
Side note: Empty string is not a valid unit of measurement. We can fix that separately.

Reply:
Please tell me how and will do this later.

Reply:
If we don't want a unit measurement we just use `None` to represent that.

Reply:
I will fix that in a future PR.
I think I have implemented all requests, if anything is needed let me know.
Test is failing because of issue with Shelly it seems.